test/aria/widgets/wai/WaiTestSuite.js

```diff
@@ -21,7 +21,7 @@ Aria.classDefinition({
 
         this.addTests("test.aria.widgets.wai.dropdown.DropdownTestSuite");
         this.addTests("test.aria.widgets.wai.autoComplete.WaiAutoCompleteTestSuite");
-        this.addTests("test.aria.widgets.wai.datePicker.DatePickerTest");
+        this.addTests("test.aria.widgets.wai.datePicker.Suite");
```

Comment:
Note that `test.aria.widgets.wai.WaiTestSuite` is not in the Jaws test suite, so it should not contain Jaws tests.